settings.gradle

```diff
@@ -11,7 +11,17 @@ def setupProject(name, path, description) {
         gradle.ext.set('componentDescriptions', [:])
     }
 
+    // TODO instead of using global "componentDescriptions" map, attach description to "project.ext"
     gradle.ext.componentDescriptions[name.substring(1)] = description
+
+    // project(name) gives us a skeleton project that we can't set ext.* on
+    gradle.afterProject { project ->
+        // However, the "afterProject" listener iterates over every project and gives us the actual project
+        // So, once we filter for the project we care about, we can set whatever we want
+        if (project.name == name.substring(1)) {
+            project.ext.enableTaskcluster = enableTaskcluster
```

Comment:
@pocmo I think you'll really like this part: with a bit of `gradle` dancing, we can assign custom properties right on the project itself!

Reply:
@mitchhentges Nice! That's something I failed to do since there's no `Project` defined at this point. I didn't know the right Gradle idiom for that. :)

My only nit here would be to rename `enableTaskcluster` to something more task oriented like `publishLibrary` or `publishToMavenRepository` since those other modules still get build and tested on taskcluster and this may be confusing.